client/synapseJavaClient/src/main/java/org/sagebionetworks/client/SynapseClientImpl.java

```diff
@@ -4043,7 +4044,7 @@ public Count getOpenMembershipInvitationCount() throws SynapseException {
 	@Override
 	public InviteeVerificationSignedToken verifyInvitee(String membershipInvitationId) throws SynapseException {
 		String uri = MEMBERSHIP_INVITATION + "/" + membershipInvitationId + "/verification";
```

Comment:
can the URI be something like /membershipInvitation/{membershipInvitationId}/inviteeVerificationSignedToken